### ISSUE TITLE
F-Droid: Add Fastlane File Structure for App Description

### ISSUE THREAD

**Issue by wba2hi** [eclipse-kuksa/kuksa-android-companion#12]
By adding the Fastlane File Structure we can speed up the time necessary from pushing a tag until it being shown in the F-Droid App store. Furthermore it is possible to provide the corresponding data, images and screenshots to be shown on the app page. 

The file structure required can be seen here: 
- https://gitlab.com/-/snippets/1895688

More info about fastlane can be found here: 
- https://docs.fastlane.tools/actions/supply/

Also in scope of this ticket: 
- Prepare MetaData File 'org.eclipse.kuksa.companion.yaml' as described here: https://gitlab.com/fdroid/fdroiddata/blob/master/CONTRIBUTING.md#metadata-preparation-without-fdroidserver
- see: https://f-droid.org/docs/Build_Metadata_Reference/

**Comment by wba2hi** [comment]
Ticket is (softly) blocked by #14 (not mandatory, but nice to have)
Ticket is blocked by #15 (package is used for MetaData file) 
Ticket is blocked by #16 (required for version and changelog)
Ticket is blocked by #17
